google/resource_compute_instance.go

```diff
@@ -1003,16 +1006,15 @@ func resourceComputeInstanceRead(d *schema.ResourceData, meta interface{}) error
 		return fmt.Errorf("Expected %d disks, API returned %d", expectedDisks, len(instance.Disks))
```

Comment:
This might be outside the scope of this PR but if we add or remove a disk outside of Terraform, then this line fails preventing the user from updating the instance.

Is this a behavior we want to fix?